themes/navy/source/css/_partial/index.styl

```diff
@@ -45,6 +45,39 @@
   @media mq-normal
     display: block
 
+#intro-news-list
+  background-color: #10171e
+  margin: -1px 0
+
+#intro-news-flex
+  display: flex
+  flex-flow: column
+  @media mq-normal
+    flex-flow: row wrap
+
+.intro-news-wrap
+  padding: 36px 30px
+  text-decoration: none
+  &:hover
+    background-color: #25a1f0
```

Comment:
recommend to use existing variable "color-link-hover".

Reply:
Updated.